Application Lifecycle Management/instances-api.md

```diff
@@ -2956,3 +2957,44 @@ ElasticBox-Release: 4.0
   "method":"cancel_import"
 }
 ```
+
+## PUT /services/instances/{instance_id}/abort_operation
+
+Abort a processing operation executing scripts on an instance when you give its ID.
```

Comment:
I would add the preconditions: write access to the instance? it is in processing state and executing a script?

Reply:
Done